.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: Publish to NPM
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v3
+        with:
+          version: 8
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "20.x"
+          cache: "pnpm"
+      - name: Install dependencies and build
+        run: pnpm install --frozen-lockfile && pnpm run build
+      - name: Publish package to NPM
+        run: npm publish
```

Comment:
Not sure, maybe we need to add `--access public`, at least that was necessary for the lingual cli workflow because by default scoped packages are published as private apparently (or that was at least the error I got back then). But I don't know how it works if the package was already published previously. (I did publish it with `--access public` before).